aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -399,8 +410,8 @@ public AwsProperties() {
 
     this.glueCatalogId = null;
     this.glueCatalogSkipArchive = GLUE_CATALOG_SKIP_ARCHIVE_DEFAULT;
+    this.glueCatalogSkipNameValidation = GLUE_CATALOG_SKIP_NAME_VALIDATION_DEFAULT;
     this.glueLakeFormationEnabled = GLUE_LAKEFORMATION_ENABLED_DEFAULT;
-
```

Comment:
nit: should not remove empty lines if not necessary, easy to cause merge conflict for other people's PRs